src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml

```diff
@@ -59,6 +76,24 @@
                 <Setter Property="FocusVisualStyle" Value="{x:Null}"/>
                 <Setter Property="ScrollViewer.PanningMode" Value="VerticalFirst"/>
                 <Setter Property="Stylus.IsFlicksEnabled" Value="False"/>
+                <Setter Property="MinWidth" Value="11" />
+                <Setter Property="HorizontalAlignment" Value="Stretch" />
+                <Style.Triggers>
+                    <MultiDataTrigger>
+                        <MultiDataTrigger.Conditions>
+                            <Condition Binding="{Binding Path=Text, RelativeSource={RelativeSource Self}}" Value="{StaticResource Empty}"/>
+                            <Condition Binding="{Binding RelativeSource={RelativeSource AncestorType={x:Type controls:DateTimePicker}}, Path=IsMouseOver}" Value="True"/>
+                        </MultiDataTrigger.Conditions>
+
+                        <Setter Property="Background" Value="{StaticResource CheckersBrush}"/>
+                        <Setter Property="BorderThickness" Value="0 0 0 2"/>
+                        <Setter Property="BorderBrush" Value="LightGray"/>
```

Comment:
@ceo1647 , don't we have special gray brushes in orc.theming for this as well?

Reply:
checkers brush is another static resource that depends on our theming
![image](https://user-images.githubusercontent.com/5163834/124439665-5bc42400-dd82-11eb-9119-2e08ed32df99.png)